app/src/main/java/ml/docilealligator/infinityforreddit/adapters/CommentsRecyclerViewAdapter.java

```diff
@@ -736,44 +736,28 @@ public void onFetchMoreCommentSuccess(ArrayList<Comment> topLevelComments,
 
                                     @Override
                                     public void onFetchMoreCommentFailed() {
-                                        if (parentPosition < mVisibleComments.size()
-                                                && parentComment.getFullName().equals(mVisibleComments.get(parentPosition).getFullName())) {
```

Comment:
I suspect there was a bug here if some comments above collapsed and placeholder's position became equal to `parentPosition`. Because this didn't check placeholder type, the check would succeed and later crash with NPE